index.html

```diff
@@ -70,7 +70,7 @@ <h3>Official Apple Material</h3>
         </header>
         <p>
         <h4>Guides/References</h4>
-          <a href="https://support.apple.com/guide/deployment/whats-new-dep950aed53e/web" target="" name="">What’s new in Apple platform deployment</a><br />
+          <a href="https://support.apple.com/guide/deployment/whats-new-dep950aed53e/web" target="" name="">What’s new in Apple platform deployment</a><br />

```

Comment:
GitHub Web editor said something about inconsistent line endings at load time, and I'm assuming this line was the offender, and it was auto-fixed. I can checkout the branch locally and revert this change if you prefer.